app/forms/declaration/TaricCodeFirst.scala

```diff
@@ -35,12 +35,13 @@ object TaricCodeFirst extends DeclarationPage {
   implicit val format = Json.format[TaricCodeFirst]
 
   val hasTaricCodeKey = "hasTaric"
+  val none = TaricCodeFirst(None)
```

Comment:
Nit: not sure what value this serves, looked cleaner with the literal value in the pattern match below as before

Reply:
It's a sort of static value and it's used in a few places, app and tests.

Reply:
:+1: